src/mocks/handlers.ts

```diff
@@ -24,7 +24,7 @@ export const handlers = [
     return res(ctx.status(200), ctx.json(storageStockPosition));
   }),
 
-  rest.get(`${apiUrl}/api/StatusHeader`, (_, res, ctx) => {
+  rest.get(`${apiUrl}/api/statusHeader?currentUtcDateTimeOverride`, (_, res, ctx) => {
```

Comment:
Is this extension needed for it to work? When I was testing it looked like it was not needed?